### ISSUE TITLE
repeated line in sticky scroll

### ISSUE THREAD

**Issue by heartacker** [microsoft/vscode#159136]
Does this issue occur when all extensions are disabled?: Yes/No

版本: 1.71.0-insider (user setup)
提交: 7ed02a69658fc07f08740b98fa0d8d96c1a61ed4
日期: 2022-08-24T05:18:03.269Z
Electron: 19.0.12
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.25182
沙盒化: Yes

Steps to Reproduce:

![image](https://user-images.githubusercontent.com/1876302/186578296-a3940ff9-6a4d-411c-a58e-8b1ecf3703e8.png)

**Comment by aiday-mar** [comment]
Hi thanks for filing the bug. Could you copy paste here the file that you have had the issue on, I will try to reproduce the error. Thanks.

**Comment by heartacker** [comment]
```
{
    // 使用 IntelliSense 了解相关属性。
    // 悬停以查看现有属性的描述。
    // 欲了解更多信息，请访问: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Python: Current File",
            "type": "python",
            "request": "launch",
            "program": "${file}",
            "console": "internalConsole",
            "justMyCode": true,
            "cwd": "${workspaceFolder}",
            "presentation": {
                "hidden": false,
                "group": "Python",
                "order": 1
            }
        },
    ],
}
```

**Comment by aiday-mar** [comment]
Hi thanks for the copy paste. I just tested on my computer, and it is working as expected as shown below on the Insiders version. Have you installed any extensions which contain a document symbol provider (language server) for JSON documents? VS Code has its own integrated language server for VS Code which I am using (ie: I have no additional extensions for JSON). 

![sticky-scroll-json](https://user-images.githubusercontent.com/61460952/186605049-e66f3041-f99f-4060-b599-59fe65c7ec93.gif)

**Comment by heartacker** [comment]
Hi， this is `launch.json` at `.vscode` folder

**Comment by aiday-mar** [comment]
A PR was proposed that should in particular solve this issue. If you would like to test, you can do so on Insiders once the PR is merged into main.
